addons/web_editor/static/src/js/wysiwyg/widgets/media.js

```diff
@@ -594,7 +626,6 @@ var FileWidget = SearchableMediaWidget.extend({
     _addData: function () {
         var self = this;
         var uploadMutex = new concurrency.Mutex();
-        var optimizeMutex = new concurrency.Mutex();
```

Comment:
That mutex was needed for optimize on upload which is not there anymore, is that it?

Reply:
Yes